terraform/README.md

```diff
@@ -151,10 +152,11 @@ pds_shared_logs_bucket_name                  = "pds-logs-dev"
 pds_nucleus_default_airflow_dag_id      = "pds-basic-registry-load-use-case"
 pds_nucleus_s3_backlog_processor_dag_id = "pds-nucleus-s3-backlog-processor"
  
-cognito_user_pool_id              = "us-west-2_ABCDEFG"
-cognito_user_pool_domain          = "pds-registry"
-auth_alb_listener_certificate_arn = "arn:aws:acm:us-west-2:123456789:certificate/ca123456-abcd-abcd-1234-abcdefghi"
-aws_elb_account_id_for_the_region = "797873946194"
+cognito_user_pool_id               = "us-west-2_ABCDEFG"
+cognito_user_pool_domain           = "pds-registry"
+auth_alb_listener_certificate_arn  = "arn:aws:acm:us-west-2:123456789:certificate/ca123456-abcd-abcd-1234-abcdefghi"
+nucleus_cloudfront_origin_hostname = "pds-sit.mcp.nasa.gov"
+aws_elb_account_id_for_the_region  = "797873946194"
```

Comment:
Glad detect-secrets didn't flag this account ID! It is absolutely public and well-known.

Reply:
> Glad detect-secrets didn't flag this account ID! It is absolutely public and well-known.

@nutjob4life , Yes, that's great!